package.json

```diff
@@ -24,5 +35,11 @@
     "gulp-nodemon": "^2.2.1",
     "gulp-rename": "^1.2.2",
     "nodemon": "^1.11.0"
+  },
+  "jest": {
+    "moduleNameMapper": {
+      "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
```

Comment:
Was boilerplate code, will remove unnecessary ones, and we can add if needed going forward.